docs/docs/.vuepress/config.js

```diff
@@ -18,195 +19,8 @@ module.exports = {
       },
     ],
   ],
-  themeConfig: {
-    algolia: {
-      appId: 'XY28M447C5',
-      apiKey: "9d9169458128b3d60c22bb04da4431c7",
-      indexName: "pipedream",
-      algoliaOptions: {
-        facetFilters: ['version:latest']
-      }
-    },
-    searchPlaceholder: "Search...",
-    logo: "/pipedream.svg",
-    repo: "PipedreamHQ/pipedream",
-    nav: [
-      {
-        text: "v2",
-        className: "docs-version",
-        ariaLabel: "Docs Version Menu",
-        items: [
-          {
-            text: "v2",
-            link: "https://pipedream.com/docs",
-            internal: true,
-            badge: "New",
-            badgeVariation: "primary",
-          },
-          {
-            text: "v1",
-            internal: true,
-            link: "https://pipedream.com/docs/v1",
-          },
-        ],
-      },
-    ],
-    smoothScroll: false,
-    // if your docs are not at the root of the repo:
-    docsDir: "docs/docs",
-    editLinks: true,
-    // custom text for edit link. Defaults to "Edit this page"
-    editLinkText: "Help us improve this page! Submit an edit on Github",
-    sidebar: [
-      "/",
-      "/quickstart/",
-      {
-        title: "Workflows",
-        children: 
-        [
-          "/workflows/",
```

Comment:
Some of these were removed, too. e.g. now there's no way to find the Environment Variables doc unless I search for it. 

I am definitely for keeping some of these sections light btw. But I still think all of these docs should live somewhere discoverable, even if it's in an advanced section.

I don't think we need to be timid about having large left navs when you expand them. We have a ton of good content we should profile! Stripe's docs have a ton of links there:

<img width="275" alt="Screen Shot 2022-03-20 at 3 41 40 PM" src="https://user-images.githubusercontent.com/242668/159189164-1e464f34-3328-4f8a-a96a-affad4048021.png">